### PR TITLE
Workspace UI

### PR DESCRIPTION
**Pull request by mahatoankitkumar**
Workspace UI

### CODE REVIEW

File: crates/frontend/src/app.rs [hunk 1]
```diff
@@ -237,6 +238,14 @@ pub fn app(app_envs: Envs) -> impl IntoView {
                                 }
                             />
 
+                            <Route
+                                ssr=SsrMode::Async
+                                path="/:tenant/workspace"
```
Comment (Datron) [inline crates/frontend/src/app.rs:243]:
```suggestion
                                path="/:org/workspace"
```

File: crates/frontend/src/api.rs [hunk 2]
```diff
@@ -276,3 +278,21 @@ pub async fn fetch_types(
...
+
+pub async fn fetch_workspaces(
+    filters: &PaginationParams,
+) -> Result<PaginatedResponse<Workspace>, ServerFnError> {
+    let client = reqwest::Client::new();
+    let host = use_host_server();
+    let url = format!("{}/workspaces?{}", host, filters.to_string());
+    let response: PaginatedResponse<Workspace> = client
+        .get(url)
+        .header("x-tenant", "dev")
```
Comment (Datron) [inline crates/frontend/src/api.rs]: @mahatoankitkumar you can remove this and add /workspace to tenant-middleware-exclusion-list env. nab

File: crates/frontend/src/app.rs (hunk 1 above)
Comment (mahatoankitkumar) [inline crates/frontend/src/app.rs:243]: Have to do this till the tenant middleware is fixed
```suggestion
                                path="/workspace"
```

Comment (Datron) [reply crates/frontend/src/app.rs:243]: Okay that works, we can also add this to exclusion list too

**Review by Datron — APPROVED** [review]

**Pull request merged**